pennylane/_qubit_device.py

```diff
@@ -568,7 +581,7 @@ def generate_samples(self):
         rotated_prob = self.analytic_probability()
 
         samples = self.sample_basis_states(number_of_states, rotated_prob)
-        return QubitDevice.states_to_binary(samples, self.num_wires)
+        return self.states_to_binary(samples, self.num_wires)
```

Comment:
This line led to `DefaultQubitJax` having an own implementation of `generate_samples` but not using it! 
Does that have any implications? I only found it because CodeCov suddenly complained about the `DefaultQubitJax` version not being covered...

Reply:
This is adressed in #2809.